examples/composite_template/ex_menu_button/imp.rs

```diff
@@ -26,25 +26,21 @@ impl ObjectSubclass for ExMenuButton {
     }
 }
 
-impl ObjectImpl for ExMenuButton {
-    fn constructed(&self, obj: &Self::Type) {
-        self.parent_constructed(obj);
-
-        let popover = &*self.popover;
-        self.toggle
-            .connect_toggled(glib::clone!(@weak popover => move |toggle| {
-                if toggle.is_active() {
-                    popover.popup();
-                }
-            }));
-
-        let toggle = &*self.toggle;
-        self.popover
-            .connect_closed(glib::clone!(@weak toggle => move |_| {
-                toggle.set_active(false);
-            }));
+#[gtk::composite_template_callbacks]
```

Comment:
```suggestion
#[gtk::template_callbacks]
```

Would call it something like this, but no strong opinions. The current name is just way too long :) any ideas @BrainBlasted @YaLTeR @Hofer-Julian ?

Reply:
`#[gtk::template_callbacks]` sounds nice

Reply:
+1 for `#[gtk::template_callbacks]`